### ISSUE TITLE
Compile failed

### ISSUE THREAD

**Issue by zel2023** [tuhh-softsec/vul4j#25]
I have successfully installed vul4j. I have also installed JAVA7, JAVA8, and MAVEN3, but when I run “vul4j compile -d /tmp/vul4j/VUL4J-10”, it still says “Compile failed”.

**Comment by Asurce** [comment]
Hi

Have you checked the `compile.log` inside the `VUL4J-10/VUL4J` directory? It might give you some hints about the issue.
Also make sure you have set the correct java paths in the environment variables or `config.py` file (you need to run the setup script again after modifying the file).

**Comment by zel2023** [comment]
Thank you for your response. I have ensured that the Java path in the config.py file has been modified. Additionally, I checked the compile.log file, which initially showed the message 'Invalid maximum heap size: -Xmx4g'. I then changed '-Xmx4g' to '-Xmx1g' in the config.py and attempted to compile again, but it still failed. Upon checking the compile log file, I found the following error message.
![微信图片_20240627224345](https://github.com/tuhh-softsec/vul4j/assets/122333859/30bead11-5fbe-42ca-b8bf-054a452ac8a9)
